src/index.d.ts

```diff
@@ -1,6 +1,13 @@
 import { IPFSContent, Token } from "@gatsby-tv/types";
 
 declare global {
+  declare namespace NodeJS {
+    interface ProcessEnv {
+      NODE_ENV: "development" | "staging" | "production";
```

Comment:
Convert this to an enum, shouldn't use string literals for comparison when possible since can lead to devs creating bugs with typos.

Reply:
In TypeScript, string literals can be used as types. If you were to provide a string value that doesn't match the listed values specified as the type, TypeScript will complain. I would say that an enum would be effective if either there were many possible values or if the values themselves are unintuitive.

Reply:
Otherwise, this is equivalent to having an enum.